creating_packages.rst

```diff
@@ -0,0 +1,18 @@
+.. _packaging:
```

Comment:
I suggest leaving this section named as "packaging", so it is easier to review. Otherwise, it seems that everything is new, I can't see changes. This rename can be done in a further PR, or leave it, the name "packaging" is not that bad.

Reply:
Packaging is not a bad name. However I think it is easier and intuitive to find files that have same name as the header. If a section is called "Creating packages" I would expect the content to be in a file with a similar name.

Reply:
Yes, I was thinking also of so many broken links. Everytime we link a docs page from github pointing to that "packaging" section, will be broken. But I am fine with the change too, no big issue